README.md

```diff
@@ -1,6 +1,7 @@
 # trezor-link
 
-[![Build Status](https://travis-ci.org/trezor/trezor-link.svg?branch=master)](https://travis-ci.org/trezor/trezor-link) [![gitter](https://badges.gitter.im/trezor/community.svg)](https://gitter.im/trezor/community)
+[![Build Status](https://github.com/trezor/trezor-link/actions/workflows/tests.yml/badge.svg)](https://github.com/trezor/trezor-link/actions/workflows/tests.yml)
+[![NPM](https://img.shields.io/npm/v/trezor-link.svg)](https://www.npmjs.org/package/trezor-link)
```

Comment:
This should be probably commited separately.

Reply:
why? this is scoped with replacing travis badges with github action badges

Reply:
Sorry, I should have been clearer about what I mean.

>  this is scoped with replacing travis badges with github action badges

The replacement of build status badge is.

Removal of gitter badge and addition of NPM badge isn't. This addition and removal probably have a different reason thean the Travis/GitHub replacement. Ideally this commit only replaces the "build status" badge and then there's another commit with a message in the sense of
```
Update badges in readme

We do not use Gitter anymore. Remove its badge.
Add badge NPM badge for quick navigation to NPM.
```

Reply:
restoring gitter badge, feel free to remove it in separated commit in the future